deepcell_toolbox/utils_test.py

```diff
@@ -34,7 +34,6 @@
 from skimage.measure import label
 
 import pytest
-
```

Comment:
Add the whitespace back in. It's good to visually separate 3rd parth resource from local ones.

Reply:
Done